src/Bicep.Core.UnitTests/Assertions/DiagnosticAssertions.cs

```diff
@@ -15,13 +15,13 @@ namespace Bicep.Core.UnitTests.Assertions
 {
     public static class DiagnosticExtensions 
```

Comment:
[nit] rename to `IDiagnosticAssertions`

Reply:
These are not extensions for Bicep Diagnostic, but are for OmniSharp Diagnostics which have different properties. I'll rename to Omnisharp.